src/Simplex/Messaging/Crypto.hs

```diff
@@ -0,0 +1,146 @@
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Simplex.Messaging.Crypto
+  ( PrivateKey (..),
+    PublicKey (..),
+    Signature (..),
+    generateKeyPair,
+    sign,
+    verify,
+    serializePrivKey,
+    serializePubKey,
+    parsePrivKey,
+    parsePubKey,
+    privKeyP,
+    pubKeyP,
+  )
+where
+
+import Crypto.Hash.Algorithms (SHA256 (..))
+import Crypto.Number.Generate (generateMax)
+import Crypto.Number.Prime (findPrimeFrom)
+import Crypto.Number.Serialize (i2osp, os2ip)
+import qualified Crypto.PubKey.RSA as C
+import qualified Crypto.PubKey.RSA.PSS as PSS
+import Data.Attoparsec.ByteString.Char8 (Parser)
+import qualified Data.Attoparsec.ByteString.Char8 as A
+import Data.ByteString.Base64
+import Data.ByteString.Char8 (ByteString)
+import Database.SQLite.Simple as DB
+import Database.SQLite.Simple.FromField
+import Database.SQLite.Simple.Internal (Field (..))
+import Database.SQLite.Simple.Ok (Ok (Ok))
+import Database.SQLite.Simple.ToField (ToField (..))
+import Simplex.Messaging.Parsers (base64P)
+import Simplex.Messaging.Util (bshow, (<$$>))
+
+newtype PublicKey = PublicKey C.PublicKey deriving (Eq, Show)
```

Comment:
`C` refers to this module in other places, is it possible to `import qualified Crypto.PubKey.RSA as RSA`?

Reply:
renamed to R